### PR TITLE
test(teams): Add a test to check if "Shared with team" allows chatting

### PR DESCRIPTION
**Pull request by nickvergessen**
### 🚧 Tasks

- [x] Add test to show the problem
- [x] Requires https://github.com/nextcloud/circles/issues/1575

### 🏁 Checklist

- [x] ⛑️ Tests (unit and/or integration) are included or not possible
- [x] 📘 API documentation in `docs/` has been updated or is not required
- [x] 🔖 Capability is added or not needed

### CODE REVIEW

File: .github/workflows/integration-oci.yml [hunk 1]
```diff
@@ -105,6 +106,13 @@ jobs:
           path: apps/call_summary_bot
           ref: ${{ matrix.call-summary-bot-versions }}
 
+      - name: Checkout circles app
+        uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
```
Comment (SystemKeeper) [inline .github/workflows/integration-oci.yml:110]: Any reason to use a different checkout version here then for the other checkouts?

File: .github/workflows/integration-pgsql.yml [hunk 2]
```diff
@@ -96,6 +97,13 @@ jobs:
           path: apps/call_summary_bot
           ref: ${{ matrix.call-summary-bot-versions }}
 
+      - name: Checkout circles app
+        uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
```
Comment (SystemKeeper) [inline .github/workflows/integration-pgsql.yml:101]: Same as above

File: .github/workflows/integration-sqlite.yml [hunk 3]
```diff
@@ -84,6 +85,13 @@ jobs:
           path: apps/call_summary_bot
           ref: ${{ matrix.call-summary-bot-versions }}
 
+      - name: Checkout circles app
+        uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
```
Comment (SystemKeeper) [inline .github/workflows/integration-sqlite.yml:89]: Same as above

File: .github/workflows/integration-mysql.yml [hunk 4]
```diff
@@ -93,6 +94,13 @@ jobs:
           path: apps/call_summary_bot
           ref: ${{ matrix.call-summary-bot-versions }}
 
+      - name: Checkout circles app
+        uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
```
Comment (SystemKeeper) [inline .github/workflows/integration-mysql.yml:98]: And here

File: .github/workflows/integration-oci.yml (hunk 1 above)
Comment (nickvergessen) [reply .github/workflows/integration-oci.yml:110]: Copy pasted but others were updated since

**Review by SystemKeeper — APPROVED** [review]
Looks good!

**Pull request merged**